src/compas/scene/sceneobject.py

```diff
@@ -13,14 +13,12 @@ class SceneObject(object):
     ----------
     item : Any
         The item which should be visualized using the created SceneObject.
-    context : str, optional
-        Explicit context to pick the SceneObject from.
-        If not specified, an attempt will be made to automatically detect the appropriate context.
+
 
     Attributes
     ----------
-    ITEM_SCENEOBJECT : dict[str, dict[Type[:class:`~compas.data.Data`], Type[:class:`~compas.scene.SceneObject`]]]
-        Dictionary mapping data types to the corresponding scene objects types per visualization context.
+    item_guids : list[object]
```

Comment:
Or perhaps this attribute goes to `SceneObject` implementation of each contexts, then the name varies depends on what it is actually tracking. In compas_rhino we call it `item_guids`, in GH `item_gemetries` and in blender `item_objects`?

Reply:
why not just guids

Reply:
That was my initial attention too. But won't `guids` creates the confusion whether it means guids for itself or the guids for drawn entities of underlying item? I understand a possible distinction could be `guid` singular refers to self, `guids` plural refers to drawn entities. Another discussion I didn't want to go there yet is whether we should have guid for SceneObject as well. Anyway, maybe easier to discuss all this in tomorrow's meeting